src/show/components/ProgressBar/index.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 
 const ProgressBar = ({ color, percentage }) => {
   return (
-    <div className='w-3/4 bg-gray-200 rounded-full '>
+    <div className='w-3/4 bg-gray-200 rounded-full'>
```

Comment:
`w-3/4`? should be `w-full` and control the width from the page/component where you use this.